ui-admin/src/study/StudyContent.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useState } from 'react'
+import React, { useContext, useState } from 'react'
```

Comment:
You'll absolutely want to review this with whitespace hidden: https://github.com/broadinstitute/juniper/pull/619/files?w=1